pygfx/utils/transform.py

```diff
@@ -491,17 +484,38 @@ def __init__(
         self._matrix_view = self._matrix.view()
         self._matrix_view.flags.writeable = False
 
+        self._wrapper: RecursiveTransform = None
+
     def flag_update(self):
         """Signal that this transform has updated."""
         self.last_modified = perf_counter_ns()
+        if self._wrapper:
+            self._wrapper.flag_update()
+
+    def _set_wrapper(self, wrapper: RecursiveTransform):
+        self._wrapper = wrapper
+
+    @property
+    def reference_up(self) -> np.ndarray:
+        """The zero-tilt reference vector used for the direction setters."""
+        if self._wrapper:
+            return self._wrapper._parent_reference_up
+        return self._reference_up_view
+
+    @reference_up.setter
+    def reference_up(self, value):
+        if self._wrapper:
+            self._wrapper._parent_reference_up = value
+        else:
+            self._reference_up[:] = la.vec_normalize(value)
```

Comment:
I guess this should call the base implementation like we've done for a few other of these setters

Reply:
Done